### ISSUE TITLE
Generated Pride Flags

### ISSUE THREAD

**Issue by leynabajaj** [leynabajaj/10Sexuality#2]
Generate a pride flag based on the results someone got

**Comment by anvitha305** [comment]
I think this would be a fun additional feature, but is it based on if the user's attracted to two or more genders and has no gender preference, we give them the pan flag or something like that? It's a cute idea but if so, it'd also be good to have a disclaimer prior to the quiz to state that the user doesn't necessarily need to accept the label and can choose other terms to label themself if they'd like to just so that people don't feel like they _need_ to ascribe to the label we give them

**Comment by leynabajaj** [comment]
> I think this would be a fun additional feature, but is it based on if the user's attracted to two or more genders and has no gender preference, we give them the pan flag or something like that? It's a cute idea but if so, it'd also be good to have a disclaimer prior to the quiz to state that the user doesn't necessarily need to accept the label and can choose other terms to label themself if they'd like to just so that people don't feel like they _need_ to ascribe to the label we give them

Of course but I was thinking that instead of just generating a pan flag, we could computer generated unique flags that have the motifs of different identities depending on what is scored. For example, say of someone shows no gendered preference but they're also on the ace scale. What if we were to take all those results into account and the computer would generate something like a tricolor ace color scheme flag for example? If they were non monogamous for example too it'd take a motif from the polyam flag and etc

**Comment by anvitha305** [comment]
Yeah that'd be much cooler than just having a single flag being represented!
